tflint/README.md

```diff
@@ -0,0 +1,49 @@
+# tflint 
+
+This package is a wrapper of the ![tflint](https://github.com/terraform-linters/tflint) project, enabling it to be natively executed from the before and after hooks. The `tflint/tflint.go` uses the MPL 2.0 license, due to the license of tflint.
+
+## How to use
+
+`tflint` is now built-in in Terragrunt hooks. It behaves the same way as other hooks, but instead of executing a command, it will call the built-in functionality of `tflint`.
+
+Here's an example:
+```hcl
+terraform {
+  before_hook "before_hook" {
+    commands     = ["apply", "plan"]
+    execute      = ["tflint"]
+  }
+}
+```
+
+The `.tflint.hcl` should exist in the same folder or one of it's parents. If Terragrunt can't find a `.tflint.hcl` file, it won't execute tflint.
```

Comment:
Which "same" folder? I imagine it's at the root level.

Reply:
If no `.tflint.hcl` file is found, does it show an error? Or just exit? Is there any logging to explain what happened?